wfe/wfe.go

```diff
@@ -1618,3 +1620,52 @@ func uniqueLowerNames(names []string) []string {
 	sort.Strings(unique)
 	return unique
 }
+
+func (wfe *WebFrontEndImpl) RevokeCert(
+	ctx context.Context,
+	logEvent *requestEvent,
+	response http.ResponseWriter,
+	request *http.Request) {
+	body, _, prob := wfe.verifyPOST(ctx, logEvent, request, wfe.lookupJWK)
```

Comment:
I think its worth highlighting the reason that `wfe.lookupJWK` is being used here since its an outlier compared to the usual `wfe.verifyPOST` calls used by other endpoints.